proto/internal/temporal/server/api/matchingservice/v1/request_response.proto

```diff
@@ -329,10 +329,21 @@ message GetBuildIdTaskQueueMappingResponse {
     repeated string task_queues = 1;
 }
 
+message ForceLoadTaskQueueRequest {
+    string namespace_id = 1;
+
+    temporal.server.api.taskqueue.v1.TaskQueuePartition task_queue_partition = 2;
+}
+
+message ForceLoadTaskQueueResponse {
+    bool was_unloaded = 1;
+}
+
+
 message ForceUnloadTaskQueueRequest {
     string namespace_id = 1;
-    string task_queue = 2;
-    temporal.api.enums.v1.TaskQueueType task_queue_type = 3;
+
+    temporal.server.api.taskqueue.v1.TaskQueuePartition task_queue_partition = 2;
```

Comment:
please be consistent and use string task queue and enum task_queue_type like everything else (for force load also)